python/vowpalwabbit/pyvw.py

```diff
@@ -121,7 +229,35 @@ def format_input(key, val):
 
     def parse(self, str_ex, labelType=pylibvw.vw.lDefault):
         """Returns a collection of examples for a multiline example learner or a single
-        example for a single example learner."""
+        example for a single example learner.
+
+        Parameters
+        ----------
+
+        str_ex : str/list of str
+            string representing examples
+        labelType : integer
```

Comment:
This is not really an integer. but a kind of fake enum. Can we be more specific here?

Reply:
I will provide the list of integers and the corresponding labelType that it maps